script/lib/native_tests.py

```diff
@@ -0,0 +1,274 @@
+from __future__ import print_function
+
+import os
+import subprocess
+import sys
+
+SOURCE_ROOT = os.path.abspath(
+    os.path.dirname(os.path.dirname(os.path.dirname(__file__))))
+VENDOR_DIR = os.path.join(SOURCE_ROOT, 'vendor')
+PYYAML_LIB_DIR = os.path.join(VENDOR_DIR, 'pyyaml', 'lib')
+sys.path.append(PYYAML_LIB_DIR)
+import yaml  #pylint: disable=wrong-import-position,wrong-import-order
+
+
+class Verbosity:
```

Comment:
This is a good start but it farms out a lot of its implementation details to callers, e.g. the use of `.ge()` and `.le()` to let callers decide when to log and when to set stdout/stderr to devnull.

Might be better to promote this into a `Logger` class that hides this implementation detail, e.g. has a `.log()` method that takes a string & looks at its private state to decide if it gets printed or not

Reply:
I can't use `Logger` with Google Tests binaries, see `TestBinary#run`.
So introducing a new class just to replace two `print()` calls looks like an overkill for me =/